pkg/migration-tool/planner/slab.go

```diff
@@ -118,6 +144,48 @@ func (s *Slab) Fetch(ctx context.Context, client *utils.Client, mint, maxt int64
 	return nil
 }
 
+func (s *Slab) mergeSubSlabsToSlab(subSlabs []*utils.PrompbResponse) []*prompb.TimeSeries {
+	l := len(subSlabs)
+	s.UpdatePBarMax(s.PBarMax() + 2)
+	s.SetDescription(fmt.Sprintf("combining fetched series from %d responses", l), 1)
+	timeseries := make(map[string]*prompb.TimeSeries)
+	for i := 0; i < l; i++ {
+		ts := subSlabs[i].Result.Timeseries
+		for _, series := range ts {
+			if s, ok := timeseries[getLabelsStr(series.Labels)]; len(series.Samples) > 0 && ok {
+				l := len(s.Samples)
```

Comment:
this masks the l on line 148. Really bad practice. Give it another name

Reply:
I am sorry, that was done unknowingly.